cli/internal/helm/charts/edgeless/constellation-services/charts/join-service/values.schema.json

```diff
@@ -14,10 +14,10 @@
             "description": "Whether or not idkeydigest should be enforced during attestation on azure.",
             "type": "boolean"
         },
-        "idkeydigest": {
-            "description": "Expected idkeydigest value for Azure SNP attestation.",
+        "idkeydigests": {
+            "description": "List of expected idkeydigest values for Azure SNP attestation.",
             "type": "string",
```

Comment:
Not sure if you tested the code in a live cluster, but does this work with an array?
I would expect the type needs to become `array`.

Reply:
Yes I tested it. I think only the example is wrong. The field itself holds a string with the marshalled JSON array. Will adapt the example.

Reply:
Ah right, that's why. Thanks!